src/orquestra/quantum/trackers.py

```diff
@@ -5,19 +5,20 @@
 from typing import Dict, List, Optional, Sequence
 
 from .api.backend import QuantumBackend
```

Comment:
Shouldn't we also remove QuantumBackend?

Reply:
Yes, fixed in [d0e7533](https://github.com/zapatacomputing/orquestra-quantum/pull/66/commits/d0e75334a2444dfca23eae76df9cbcfffbf037a0)